diff -up ffmpegthumbnailer-2.0.6/libffmpegthumbnailer/filmstripfilter.cpp.orig ffmpegthumbnailer-2.0.6/libffmpegthumbnailer/filmstripfilter.cpp
--- ffmpegthumbnailer-2.0.6/libffmpegthumbnailer/filmstripfilter.cpp.orig	2011-02-13 14:44:28.892121640 +0200
+++ ffmpegthumbnailer-2.0.6/libffmpegthumbnailer/filmstripfilter.cpp	2011-02-13 14:44:47.129129279 +0200
@@ -16,6 +16,7 @@
 
 #include "filmstripfilter.h"
 #include "filmstrip.h"
+#include <stddef.h>
 
 namespace ffmpegthumbnailer
 {
